src/shared/services/banners/banners.service.ts

```diff
@@ -14,19 +14,15 @@ export class BannersService {
   }
 
   getBanners(jurisdictionReferences: string[]): Observable<Banner[]> {
-    let url = this.appConfig.getBannersUrl();
-    let headers = new HttpHeaders()
+    const url = this.appConfig.getBannersUrl();
+    const headers = new HttpHeaders()
       .set('experimental', 'true')
       .set('Accept', BannersService.V2_MEDIATYPE_BANNERS)
       .set('Content-Type', 'application/json');
     let params = new HttpParams();
```

Comment:
params could be const as well.

Reply:
That was my first reading of it too. However, the `forEach()` in the line below reassigns it each time so it needs to remain `let`. I'm not 100% sure why the reassignment is needed but it was done deliberately by @udaydenduluri33 in the first place - and is the same in some other places that `HttpHeaders` is used - so I assume it's necessary. 🤷

Reply:
The reassignment is not necessary at all. But maybe @udaydenduluri33 has an explanation?

Reply:
I remember helping @RiteshHMCTS  on this for his ticket. This is because HttpHeaders are immutable.

Reply:
Ah! That would explain it then. I hadn't appreciated that and I've just checked the Angular docs to confirm there's no alternative mechanism... but @udaydenduluri33's explanation looks correct from what I can see. Happy to resolve this conversation, @kuda1992?

Reply:
@udaydenduluri2 @paul-graham just checked out the documentation. its apparently correct. I will resolve the conversation.